nanome_msms/MSMS.py

```diff
@@ -24,6 +27,7 @@ def start(self):
         self._msms_instances = {}
         self._msms_tasks = {}
         self.create_menu()
+        self._nanome_version = version.parse(nanome.__version__)
```

Comment:
You should pin the nanome version in requirements.txt, and remove this variable from the plugin

`nanome==0.35.8`

Reply:
I wanted the plugin to still work for previous versions but removing some features